.github/actions/spell-check/expect.txt

```diff
@@ -124,8 +129,10 @@ BLURBEHIND
 BLURREGION
 bmi
 bms
+BNDBk
 BNumber
 BODGY
+bodgy
```

Comment:
Is this an actual English name added to a code or is it a username instead? I'm thinking it should be added to names.txt instead.